src/Types.hs

```diff
@@ -243,7 +256,7 @@ type CallMapParseError = ParsecErr.ParseError
 -- | An error parsing one of the processing units.  Either a C parse error or a CallMapParseError
 data ProcessingUnitParseError =
   CSourceUnitParseError CParser.ParseError
-  | CallMapUnitParseError !CallMapParseError
+  | CallMapUnitParseError String
```

Comment:
Is `CallMapParseError` now unused? We could change it:

```diff
-type CallMapParseError = ParsecErr.ParseError
+type CallMapParseError = String
```

Reply:
It seemed a bit unnecessary as the data constructor is already pretty descriptive but I have gone ahead and reintroduced the type synonym.